src/main/java/com/blamejared/crafttweaker/impl_native/util/ExpandActionResultType.java

```diff
@@ -0,0 +1,17 @@
+package com.blamejared.crafttweaker.impl_native.util;
+
+import com.blamejared.crafttweaker.api.annotations.ZenRegister;
+import com.blamejared.crafttweaker_annotations.annotations.Document;
+import com.blamejared.crafttweaker_annotations.annotations.NativeTypeRegistration;
+import net.minecraft.util.ActionResultType;
+import org.openzen.zencode.java.ZenCodeType;
+
+@ZenRegister
+@Document("vanilla/api/util/ActionResultType")
+@NativeTypeRegistration(value = ActionResultType.class, zenCodeName = "crafttweaker.api.util.ActionResultType")
+public class ExpandActionResultType {
+    @ZenCodeType.Getter("name")
+    public static String name(ActionResultType internal) {
+        return internal.name();
+    }
```

Comment:
We automatically register a `name` getter for enums, so no need for this.

https://github.com/ZenCodeLang/ZenCode/blob/ea0223ac1fd6cfca99b5d665ae66e1b2df5c7c34/JavaIntegration/src/main/java/org/openzen/zencode/java/module/converters/JavaNativeClassConverter.java#L211